src/InOneWeekend/material.h

```diff
@@ -38,38 +38,34 @@ class dielectric : public material {
 
         virtual bool scatter(
             const ray& r_in, const hit_record& rec, vec3& attenuation, ray& scattered
-        ) const  {
-            vec3 outward_normal;
-            vec3 reflected = reflect(r_in.direction(), rec.normal);
-            double ni_over_nt;
+        ) const {
             attenuation = vec3(1.0, 1.0, 1.0);
-            vec3 refracted;
-            double reflect_prob;
-            double cosine;
-
-            if (dot(r_in.direction(), rec.normal) > 0) {
-                outward_normal = -rec.normal;
-                ni_over_nt = ref_idx;
-                cosine = dot(r_in.direction(), rec.normal) / r_in.direction().length();
-                cosine = sqrt(1 - ref_idx*ref_idx*(1-cosine*cosine));
+            double etai_over_etat;
```

Comment:
Needs a better name. I don't know what this is.

Reply:
Nor I.
I hate ni_over_nt (because it _isn't_ `n` it's a greek `eta`)
we can change to `eta1_over_eta2`, but that would be a PITA
we could do something like `refractive_ratio`?

Reply:
... `refractive_ratio` would be getting away from the fact that we're actually doing quite complicated mathematics, and I don't think we should try to disguise that fact

Reply:
Oh well. Doesn't need to block this PR.